### PR TITLE
Add fade in/fade out animation on Combos page

### LINKED ISSUE #2578: Add fade in/fade out animation on Combos page
when you select and deselect heroes the images should have a short fade in/out transition effect

### PR DESCRIPTION
**Pull request by soumyankar**
Fixes #2578 

Added FadeIn/FadeOut Animations when heroes get selected/deselected from the HeroSelector.

(Apologies if I've made errors, I'm still an amateur.)

### CODE REVIEW

**Comment by Argn0** [comment]
I feel like a 1 second long animation is too slow. Also the intent was to also animate the icons in the team A and team B overview, which I don't think is possible to do with just CSS by the way.

**Comment by howardchung** [comment]
@Argn0 do you think this PR would be fine if the time was shortened? Icon animation could probably be handled separately

**Comment by atilla4455** [comment]
Yes. I think we could just make it 300ms?

**Comment by soumyankar** [comment]
Should I go ahead and make it 300ms then?

**Comment by Argn0** [comment]
I replied with another github account. Sorry about the confusion.

Yes 300ms is okay.

File: src/components/Combos/Styles.jsx [hunk 1]
```diff
@@ -81,10 +81,10 @@ export const StyledHeroSelector = styled.div`
   .ts-container.selected {
     opacity: 1;
     background-color: rgba(32, 36, 45, 0.76);
-    transition: none;
     & > div {
       display: none;
     }
+    transition: all 1s ease-out;
```
Comment (howardchung) [inline src/components/Combos/Styles.jsx:87]: what about this one?

**Review by howardchung — APPROVED** [review]

**Pull request merged**